src/v2/view-builder/views/ov/ChallengeOktaVerifyTotpView.js

```diff
@@ -13,8 +15,40 @@ const Body = BaseForm.extend(Object.assign(
     save() {
       return loc('mfa.challenge.verify', 'login');
     },
+
+    initialize: function() {
+      BaseForm.prototype.initialize.apply(this, arguments);
+      this.listenTo(this.options.appState, 'afterError', (convertedErrors) => {
+        // Need to wait until BaseForm renders the error so we can replace it
+        setTimeout(() => {
+          this._showUserVerifcationError(convertedErrors);
+        });
+      });
+    },
+
+    _showUserVerifcationError(convertedErrors) {
```

Comment:
did you try overriding `showMessages` instead of creating a new fn `_showUserVerifcationError` here ? Since you want to display this enhanced callout based on a key `OV_UV_ENABLE_BIOMETRIC_SERVER_KEY`. It just make sense to override `showMessages`

Reply:
if you look at this PR there were attempts to show this error in other ways. `showMessages` specifically does not work for this use case, because `showMessages` is only called in initialize for a BaseForm (so it works when we change from a remediation form to another). In this case we are not changing forms, we are getting an error from the API and we just need to replace the format of that error we got to show it more user friendly, that is why I need to replace it this way

Reply:
I guess we do the same for push and even there we dont rerender the form after we get back error messages I guess. 
https://github.com/okta/okta-signin-widget/blob/f53b68168988903b0d63d4c584f68ef5693a03ee/src/v2/view-builder/views/ov/ChallengeOktaVerifyResendPushView.js#L20

Reply:
Push is different, we are polling, and the poll requests are returned with http 200 status. When the poll response comes with `messages` object, it also comes with a different remediation form name. This is why `showMessages` works there, sine we move to a different form

For TOTP, there is no polling. We make the request with the code and the API returns a 400 indicating it was wrong. At this time, BaseForm renders the error messages, and what I am trying to do here is replace that error messages with a different one (better format for our UI)